### DISCUSSION TITLE
v3.3.6b0

### DISCUSSION THREAD

**Discussion by DTTerastar** [ESPresense/ESPresense#1430] [Announcements]
## Changes

- Svelte 5 upgrade @DTTerastar (#1424)
- Build improvements @DTTerastar (#1421)
- Update CODEOWNERS @DTTerastar (#1397)

## Chores

- Bump peter-evans/create-pull-request from 6 to 7 @dependabot (#1427)
- Bump tailwindcss from 3.4.14 to 3.4.15 in /ui @dependabot (#1415)
- Bump postcss from 8.4.47 to 8.4.49 in /ui @dependabot (#1410)
- Bump tailwindcss from 3.4.13 to 3.4.14 in /ui @dependabot (#1400)
- Bump svelte-preprocess from 6.0.2 to 6.0.3 in /ui @dependabot (#1395)

<hr /><em>This discussion was created from the release <a href='https://github.com/ESPresense/ESPresense/releases/tag/v3.3.6b0'>v3.3.6b0</a>.</em>

**Comment by DrSpaldo** [comment]
What is currently the best way to do a single release update for ESPresense?
